config/events/config.go

```diff
@@ -0,0 +1,43 @@
+package events
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for events group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_cloudwatch_event_permission", func(r *config.Resource) {
+		r.References["event_bus_name"] = config.Reference{
+			Type:              "github.com/upbound/provider-aws/apis/events/v1beta1.Bus",
```

Comment:
If they are in the same package, just the type name of the reference target should suffice:
```suggestion
			Type:              "Bus",
```

Reply:
Ah, I didn't know that

Thanks for the heads up!